cincinnati/src/lib.rs

```diff
@@ -128,6 +128,28 @@ impl Graph {
     pub fn releases_count(&self) -> u64 {
         self.dag.node_count() as u64
     }
+
+    /// Prune the graph from all abstract releases.ConcreteRelease
+    ///
+    /// Return the number of pruned releases
+    pub fn prune_abstract(&mut self) -> usize {
```

Comment:
Side note (not for this PR): we should document what is an abstract release, why/how they can appear as part of graph, and what's the special logic around them.